lib/backup_restore/s3_backup_store.rb

```diff
@@ -0,0 +1,90 @@
+require_dependency "backup_restore/backup_store"
+require_dependency "s3_helper"
+
+module BackupRestore
+  class S3BackupStore < BackupStore
+    DOWNLOAD_URL_EXPIRES_AFTER_SECONDS ||= 15
+    UPLOAD_URL_EXPIRES_AFTER_SECONDS ||= 21600 # 6 hours
```

Comment:
Usually for large numbers, writing it as `21_600` is much easier to read than `21600`